lib/redux/middleware/discover_middleware.dart

```diff
@@ -1,17 +1,17 @@
+import 'package:farmsmart_flutter/data/repositories/articles_directory_repository.dart';
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
-import 'package:farmsmart_flutter/data/repositories/articles_repository.dart';
 import 'package:farmsmart_flutter/redux/home/discover/discover_actions.dart';
 import 'package:redux/redux.dart';
 
 class DiscoverMiddleWare extends MiddlewareClass<AppState> {
   @override
   Future call(Store<AppState> store, dynamic action, NextDispatcher next) async {
-    if(action is FetchArticleListAction) {
-        var listOfArticles = await ArticlesRepository.get().getListOfArticles();
-        listOfArticles = await ArticlesRepository.get().getListOfArticlesWithImages(listOfArticles);
-        store.dispatch(UpdateArticleListAction(listOfArticles));
+    if(action is FetchArticleDirectoryAction) {
+        var articlesDirectory = await ArticlesDirectoryRepository.get().getDirectory();
```

Comment:
I'd like to adopt the coding standard from here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

One of the suggestion is to avoid using var. 
I think that advice applies very well in this piece of code. I couldn't see what type articlesDirectory was and it looked like it's reused in several calls (which could have changed type). 
But looking more closely in code, it's actually the same type. Please use the expected type.

Reply:
Done 👍🏻